content/v1.12/getting-started/provider-azure-part-2.md

```diff
@@ -0,0 +1,1050 @@
+kubectl get virtualmachine -n test
+NAME                     SYNCED   READY   CONNECTION-SECRET   AGE
+claimed-virtualmachine   True     True                        3m40s
+```
+
+When Crossplane creates a _Claim_, a unique _composite resource_ is also
+created. View the new _composite resource_ with `kubectl get xvirtualmachine`.
+
+```shell {copy-lines="1"}
+kubectl get xvirtualmachine
+NAME                           SYNCED   READY   COMPOSITION                             AGE
+claimed-virtualmachine-cw6cv   True     True    crossplane-quickstart-vm-with-network   3m57s
+```
+
+The _composite resource_ exists at the "cluster scope" while the _Claim_ exists
+at the "namespace scope."
+
+Create a second namespace and a second claim.
+
+```shell
+kubectl create namespace test2
+cat <<EOF | kubectl apply -f -
+apiVersion: custom-api.example.org/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: claimed-virtualmachine
+  namespace: test2
+spec:
+  region: "US"
+EOF
+```
+
+View the _claims_ in all namespaces with `kubectl get virtualmachine -A`
+
+```shell {copy-lines="1"}
+kubectl get virtualmachine -A
+NAMESPACE   NAME                     SYNCED   READY   CONNECTION-SECRET   AGE
+test        claimed-virtualmachine   True     True                        12m
+test2       claimed-virtualmachine   True     True                        7m35s
+```
+
+Now look at the _composite resources_ at the cluster scope.
+
+```shell {copy-lines="1"}
+kubectl get xvirtualmachine
+NAME                           SYNCED   READY   COMPOSITION                             AGE
+claimed-virtualmachine-7jth5   True     True    crossplane-quickstart-vm-with-network   7m53s
+claimed-virtualmachine-cw6cv   True     True    crossplane-quickstart-vm-with-network   12m
+```
+
+Crossplane created a second _composite resource_ for the second _Claim_.
+
+Looking at the virtual machines and network interfaces shows two of each 
+resource, one for each claim.
+
+```shell {copy-lines="1"}
+kubectl get linuxvirtualmachines
+NAME                                 READY   SYNCED   EXTERNAL-NAME                        AGE
+claimed-virtualmachine-7jth5-v2gsh   True    True     claimed-virtualmachine-7jth5-v2gsh   8m10s
+claimed-virtualmachine-cw6cv-w8v65   True    True     claimed-virtualmachine-cw6cv-w8v65   13m
+```
+
+```shell {copy-lines="1"}
+kubectl get networkinterface
+NAME                                 READY   SYNCED   EXTERNAL-NAME                        AGE
+claimed-virtualmachine-7jth5-hj657   True    True     claimed-virtualmachine-7jth5-hj657   8m44s
+claimed-virtualmachine-cw6cv-f9z4f   True    True     claimed-virtualmachine-cw6cv-f9z4f   13m
+```
+
+### Delete the claims
+Removing the _claims_ removes the _composite resources_ and the associated
+_managed resources_.
+
+```shell
+kubectl delete virtualmachine claimed-virtualmachine -n test
+kubectl delete virtualmachine claimed-virtualmachine -n test2
+```
+
+Verify Crossplane removed all the _managed resources_.
+
+```shell
```

Comment:
the copy button for this block and the next one below is copying the entire text, which includes the output, instead of just the command.

Reply:
fixed